### ISSUE TITLE
Performance issue: Iteration for add a elemente in a second list

### ISSUE THREAD

**Issue by mmalaquiasdev** [Yorubaname/yorubaname-website#68]
Hi,

```
 allNameEntries.forEach(nameEntry -> {
            names.add(nameEntry);
 });
```
With this code we are using a loop just for add a new element in another list, soon it's better change that for a line more simple to reading and less memory cust.

```
names.addAll(allNameEntries);
```

**Comment by mmalaquiasdev** [comment]
solved #64

**Comment by mmalaquiasdev** [comment]
Merged #69
